usagov/vote-gov-drupal
# Vote-3147: RTL style updates

## Jira ticket

[Vote-3147](https://cm-jira.usa.gov/browse/VOTE-3147)

## Description

Addressing various RTL style issues:
1. accessibility page had element off to the side causing it to zoom out in mobile 
2. the arrows for quick links were not pointing the right direction 
3. the accessibility bar was going behind the touchpoints form on the accessibility page 

<details>
<summary>Screen shots</summary>
<br>
<img width="1509" alt="Screenshot 2024-12-01 at 8 45 21 PM" src="https://github.com/user-attachments/assets/babdc10d-84c6-4695-a4d9-997dd6afc33c">
<br>
<br>
<img width="1507" alt="Screenshot 2024-12-01 at 8 44 50 PM" src="https://github.com/user-attachments/assets/c674a555-1af9-4e66-82b8-bec819552ff7">
<br>
<br>
<img width="1505" alt="Screenshot 2024-12-01 at 8 44 27 PM" src="https://github.com/user-attachments/assets/32448dbe-210f-4828-91fd-e528461bbfdd">
</details>

## Deployment and testing

### Post-deploy steps

1. cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. switch to the AR translation and verify that the arrows for quick links are pointing the right direction 
2. visit the accessibility page  and ensure that you do not need to scroll over to see the page while in mobile view 
3. while on the accessibility page scroll to the touchpoints form and check that the accessability bar does not go behind the form.

## Checklist for the Developer




## Checklist for the Peer Reviewers
